tests/filecheck/frontend/refactor/bigint.py

```diff
@@ -0,0 +1,20 @@
+# RUN: python %s | filecheck %s
+
+from xdsl.dialects.bigint import BigIntegerType
+from xdsl.frontend.pyast.context import CodeContext
+from xdsl.frontend.pyast.program import FrontendProgram
+
+p = FrontendProgram()
+p.register_type(int, BigIntegerType)
+with CodeContext(p):
+    # CHECK:      builtin.module {
+    # CHECK-NEXT:   func.func @foo(%0 : !bigint.bigint) -> !bigint.bigint {
```

Comment:
Can you write down somewhere a reminder to add the python names as hints to the ssa values? I have a feeling this is actually important for debuggability and not just a nice to have